static/app/views/settings/organizationGeneralSettings/organizationSettingsForm.tsx

```diff
@@ -48,8 +49,21 @@ function OrganizationSettingsForm({initialData, onSave}: Props) {
 
   const forms = useMemo(() => {
     const formsConfig = cloneDeep(organizationSettingsFields);
+    const organizationIdField: FieldObject = {
+      name: 'organizationId',
+      type: 'string',
+      disabled: true,
+      label: t('Organization ID'),
+      setValue(_, _name) {
+        return organization.id;
+      },
+      help: `This is the organization's unique identifier. It cannot be changed.`,
```

Comment:
this is more subjective, but I think this matches the other settings text slightly better
```suggestion
      help: `The unique identifier for this organization. It cannot be modified.`,
```

Reply:
good call, i updated the project id description as well